config/samples/aerospikerestore.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: asdb.aerospike.com/v1beta1
+kind: AerospikeRestore
+metadata:
+  labels:
+    app.kubernetes.io/name: aerospikerestore
+    app.kubernetes.io/instance: aerospikerestore-sample
+    app.kubernetes.io/part-of: aerospike-kubernetes-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: aerospike-kubernetes-operator
+  name: aerospikerestore-sample
+  namespace: aerospike
+spec:
+  backupService:
+    name: aerospikebackupservice-sample
+    namespace: aerospike
+  type: Full
+  config:
+    destination:
+      label: destinationCluster
+      credentials:
+        password: admin123
+        user: admin
+      seed-nodes:
+        - host-name: aerocluster.test.svc.cluster.local
```

Comment:
let the host-name be same in both backup and restore file.

Reply:
Backup file hostname is for source cluster in aerospike namespace and restore file hostname is for a different destination cluster in test namespace